src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+  status: '',
+});
+
+const statusReducer = (state = [], action) => {
```

Comment:
- [OPTIONAL ] Same here, Let's create  empty array to initial state eg: `const initialState = [ ];`